src/rewriter/gen_symbol_rewriter_dictionary_main.cc

```diff
@@ -76,6 +78,861 @@ ABSL_FLAG(std::string, output_string_array, "",
+      0x9d72, 0x9d7a, 0x9d87, 0x9d89, 0x9d8f, 0x9d9a, 0x9da4, 0x9da9, 0x9dab,
+      0x9daf, 0x9db2, 0x9db4, 0x9db8, 0x9dba, 0x9dbb, 0x9dc1, 0x9dc2, 0x9dc4,
+      0x9dc6, 0x9dcf, 0x9dd3, 0x9dd9, 0x9de6, 0x9ded, 0x9def, 0x9df2, 0x9df8,
+      0x9df9, 0x9dfa, 0x9dfd, 0x9e1a, 0x9e1b, 0x9e1e, 0x9e75, 0x9e78, 0x9e79,
+      0x9e7d, 0x9e7f, 0x9e81, 0x9e88, 0x9e8b, 0x9e8c, 0x9e91, 0x9e92, 0x9e93,
+      0x9e95, 0x9e97, 0x9e9d, 0x9e9f, 0x9ea5, 0x9ea6, 0x9ea9, 0x9eaa, 0x9ead,
+      0x9eb8, 0x9eb9, 0x9eba, 0x9ebb, 0x9ebc, 0x9ebe, 0x9ebf, 0x9ec4, 0x9ecc,
+      0x9ecd, 0x9ece, 0x9ecf, 0x9ed0, 0x9ed2, 0x9ed4, 0x9ed8, 0x9ed9, 0x9edb,
+      0x9edc, 0x9edd, 0x9ede, 0x9ee0, 0x9ee5, 0x9ee8, 0x9eef, 0x9ef4, 0x9ef6,
+      0x9ef7, 0x9ef9, 0x9efb, 0x9efc, 0x9efd, 0x9f07, 0x9f08, 0x9f0e, 0x9f13,
+      0x9f15, 0x9f20, 0x9f21, 0x9f2c, 0x9f3b, 0x9f3e, 0x9f4a, 0x9f4b, 0x9f4e,
+      0x9f4f, 0x9f52, 0x9f54, 0x9f5f, 0x9f60, 0x9f61, 0x9f62, 0x9f63, 0x9f66,
+      0x9f67, 0x9f6a, 0x9f6c, 0x9f72, 0x9f76, 0x9f77, 0x9f8d, 0x9f95, 0x9f9c,
+      0x9f9d, 0x9fa0, 0xff01, 0xff03, 0xff04, 0xff05, 0xff06, 0xff08, 0xff09,
+      0xff0a, 0xff0b, 0xff0c, 0xff0e, 0xff0f, 0xff10, 0xff11, 0xff12, 0xff13,
+      0xff14, 0xff15, 0xff16, 0xff17, 0xff18, 0xff19, 0xff1a, 0xff1b, 0xff1c,
+      0xff1d, 0xff1e, 0xff1f, 0xff20, 0xff21, 0xff22, 0xff23, 0xff24, 0xff25,
+      0xff26, 0xff27, 0xff28, 0xff29, 0xff2a, 0xff2b, 0xff2c, 0xff2d, 0xff2e,
+      0xff2f, 0xff30, 0xff31, 0xff32, 0xff33, 0xff34, 0xff35, 0xff36, 0xff37,
+      0xff38, 0xff39, 0xff3a, 0xff3b, 0xff3d, 0xff3e, 0xff3f, 0xff40, 0xff41,
+      0xff42, 0xff43, 0xff44, 0xff45, 0xff46, 0xff47, 0xff48, 0xff49, 0xff4a,
+      0xff4b, 0xff4c, 0xff4d, 0xff4e, 0xff4f, 0xff50, 0xff51, 0xff52, 0xff53,
+      0xff54, 0xff55, 0xff56, 0xff57, 0xff58, 0xff59, 0xff5a, 0xff5b, 0xff5c,
+      0xff5d, 0xffe3, 0xffe5,
+  };
+
+  Char16BitMap map;
+  for (char16_t codepoint : kJisX0208Chars) {
+    map.set(codepoint);
+  }
+
+  // The following characters need to be allow-listed to be fully compatible
+  // with the legacy Util::IsJisX0208() behavior.
+
+  // All 7-bit ASCII characters:
+  for (char16_t codepoint = 0x0000; codepoint <= 0x007f; ++codepoint) {
+    map.set(codepoint);
+  }
+
+  // HALFWIDTH characters:
+  // U+FF61: "｡" HALFWIDTH IDEOGRAPHIC FULL STOP
+  // U+FF62: "｢" HALFWIDTH LEFT CORNER BRACKET
+  // U+FF63: "｣" HALFWIDTH RIGHT CORNER BRACKET
+  // U+FF64: "､" HALFWIDTH IDEOGRAPHIC COMMA
+  // U+FF65: "･" HALFWIDTH KATAKANA MIDDLE DOT
+  // U+FF66: "ｦ" HALFWIDTH KATAKANA LETTER WO
+  // U+FF67: "ｧ" HALFWIDTH KATAKANA LETTER SMALL A
+  // ...
+  // U+FF9D: "ﾝ" HALFWIDTH KATAKANA LETTER N
+  // U+FF9E: "ﾞ" HALFWIDTH KATAKANA VOICED SOUND MARK
+  // U+FF9F: "ﾟ" HALFWIDTH KATAKANA SEMI-VOICED SOUND MARK
+  for (char16_t codepoint = 0xff61; codepoint <= 0xff9f; ++codepoint) {
+    map.set(codepoint);
+  }
+
+  // Misc characters:
+  map.set(0x00a5); // "\" YEN SIGN
```

Comment:
Please add two whitespaces before the comment.

```
map.set(0x00a5);  // "\" YEN SIGN
map.set(0x203e);  // "‾" OVERLINE
```

Reply:
Done.